### PR TITLE
House: Added params to address Issue #565 which is referenced in Issu…

### LINKED ISSUE #565: house: add parameters to exercise placeholder
We would like you to help us add expected parameters in placeholder templates as described in #509 to make it easier for learners to start working on the exercise in question.

Feel free to ask any questions, we'll be glad to help!

### PR DESCRIPTION
**Pull request by atg-abhishek**
…e #509

Resolves #565 // m-a-ge

### CODE REVIEW

File: exercises/house/house.py [hunk 1]
```diff
@@ -1,6 +1,6 @@
-def verse():
+def verse(verse_text):
```
Comment (m-a-ge) [inline exercises/house/house.py]: You suggestion does not correlate with https://github.com/exercism/python/blob/master/exercises/house/house_test.py

Could you please clarify?

**Comment by N-Parsons** [comment]
@atg-abhishek, are you still working on this?

**Comment by N-Parsons** [comment]
I'm going to close this due to inactivity, but if you would like to continue working on it, feel free to reopen.

**Comment by atg-abhishek** [comment]
Hi sorry for the slow response, no unfortunately I wasn't able to work on
it!

**Pull request closed without merging**